*** tests/A-04-01-M-ok.out	2018-11-05 16:46:51.000000000 +0000
--- tests/A-04-01-M-ok.outhyp	2018-11-10 16:29:13.982335611 +0000
***************
*** 9,16 ****
  3 - Mostrar pessoas
  4 - Procurar pessoa
  0 - Sair
! Escolha uma opção: Número de telefone: FUNCIONÁRIO|101010|123456789|José Carlos
! Portal Pessoal
  1 - Mostrar pessoa
  2 - Alterar número de telefone
  3 - Mostrar pessoas
--- 9,15 ----
  3 - Mostrar pessoas
  4 - Procurar pessoa
  0 - Sair
! Escolha uma opção: Número de telefone: Portal Pessoal
  1 - Mostrar pessoa
  2 - Alterar número de telefone
  3 - Mostrar pessoas
